src/main/java/org/prebid/server/settings/model/Account.java

```diff
@@ -25,9 +26,13 @@ public class Account {
 
     AccountHooksConfiguration hooks;
 
+    @JsonIgnore
+    boolean empty;
+
     public static Account empty(String id) {
```

Comment:
This is strange. Every time you want to override field in account object you need to set empty flag to false. Whats even worse you need to remember about this. Please, take a look at how this is done in `com.iab.openrtb.request.Content` and do it the same way.

Reply:
@And1sS it's not the same since an empty account has an id for some reason, but account is immutable, so once it's empty, it's empty

Reply:
> @And1sS it's not the same since an empty account has an id for some reason, but account is immutable, so once it's empty, it's empty.

Yes, and when you want to change something via rebuilding existing object, you still need to clear this flag. Or set it, if you cleared some

Reply:
Ok, since account contains id, lets do something like this:

```java
public boolean isEmpty() {
        return this.equals(empty(id));
}
```